.gitignore

```diff
@@ -11,6 +11,4 @@ dbatools.psprojs
 # ignore library
 bin/dbatools.dll
 bin/dbatools.xml
-bin/dbatools.pdb
-bin/projects/dbatools/dbatools/obj/*
-bin/projects/dbatools/*.user
```

Comment:
If someone starts using the VS Project included in the repository the files generated on a build would get picked up by removing line 15 and 16. Is that intentional?